user_data/strategies/Strategy002.py

```diff
@@ -11,18 +11,18 @@
 import freqtrade.vendor.qtpylib.indicators as qtpylib
 import numpy # noqa
 from datetime import datetime
+import threading
 
-from wao.util import execute, back_test
+from wao._429_file_util import create_429_directory, setup_429
 from wao.config import Config
-from user_data.strategies.notifier import send_start_deliminator_message
 from wao.strategy_controller import StrategyController
 
 
 class Strategy002(IStrategy):
     controller = StrategyController()
+    controller.send_starter_message('Freq Strategy002 ')
+    setup_429()
```

Comment:
we can move this in StrategyController constructor as well

Reply:
do you mean to move the setup_429() to the controllers?

Reply:
Yes.
Always refer to the line number where the comment is added.